core/cmd/jobs_commands_test.go

```diff
@@ -323,6 +323,8 @@ func TestClient_DeleteJobV2(t *testing.T) {
 
 	requireJobsCount(t, app.JobORM, 1)
 
+	time.Sleep(5 * time.Second)
```

Comment:
Is this meant to be left in? If so I'd want a comment to justify sleeping 5 seconds!

Reply:
Dropped it to 3. This is to ensure that the test doesn't start flaking at some point because the job.Spawner takes too long to activate the job after it's been added to the DB.

Reply:
Changed this so that you can directly wait on the job.Spawner activating a job after it's added to the DB